modular_chomp/code/modules/projectiles/ammo_refactor/ammo_overrides.dm

```diff
@@ -0,0 +1,834 @@
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/rifle/a762
+
+/obj/item/weapon/gun/projectile/shotgun/pump/rifle/lever/trailgun
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/rifle/a762
+
+/obj/item/weapon/gun/projectile/revolver
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/a357
+
+/obj/item/weapon/gun/projectile/revolver/lemat
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/a38
+
+/obj/item/weapon/gun/projectile/revolver/stainless
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/a357
+
+/obj/item/weapon/gun/projectile/revolver/detective
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/a38
+
+/obj/item/weapon/gun/projectile/revolver/detective45
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/pistol/rubber
+
+/obj/item/weapon/gun/projectile/revolver/lombardi
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/a357
+
+/obj/item/weapon/gun/projectile/revolver/lombardi/panther
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/a357
+
+/obj/item/weapon/gun/projectile/revolver/lombardi/gold
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/a357
+
+/obj/item/weapon/gun/projectile/revolver/cappeacekeeper
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/pistol/strong
+
+/obj/item/weapon/gun/projectile/revolver/mateba
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/a357
+
+/obj/item/weapon/gun/projectile/revolver/deckard
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/a38
+
+/obj/item/weapon/gun/projectile/revolver/deckard/emp
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/ion/small
+
+/obj/item/weapon/gun/projectile/revolver/webley
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/pistol/strong
+
+/obj/item/weapon/gun/projectile/revolver/consul
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/pistol/rubber/strong
+
+/obj/item/weapon/gun/projectile/revolver/nova
+	caliber = "small"
+	ammo_type = /obj/item/ammo_casing/simple/small
+	projectile_type = /obj/item/projectile/bullet/a357
```

Comment:
I'm gonna fucking kill someone.
This isn't your work but this is where I **would have** started with a gun refactor PR. Why the hell does a357 have _its own projectile type_???? It sure shouldn't! I already made code for that years ago. It is a _medium_ strength bullet projectile. I already made code and left comments explaining it and yet it gets ignored and people keep making snowflake code I am gonna keelhaul them aaaaaaaaaaaaaaa

unless I'm delusional and never actually made those changes, in which case, uh, oops